pkg/kubernetes/pod/v1alpha1/pod_test.go

```diff
@@ -31,6 +31,16 @@ func fakeRunningAPIPodObject(podNames []string) []v1.Pod {
 	return plist
 }
 
+func fakeAPIPodObject(podNames []string) []v1.Pod {
```

Comment:
Can we rename function to `fakeAPIPodList`

Reply:
`fakeAPIPodList` is already used to test `WithAPIList` function

Reply:
Yes we can reuse the `fakeAPIPodList` to test `WithAPIObject`. Should I change that ?

Reply:
yes

Reply:
yes

Reply:
Done sir